pkg/synthetictests/duplicated_events.go

```diff
@@ -185,6 +185,11 @@ var knownEventsBugs = []knownProblem{
 		BZ:       "https://bugzilla.redhat.com/show_bug.cgi?id=2017435",
 		Topology: topologyPointer(v1.SingleReplicaTopologyMode),
 	},
+	// builds tests trigger many changes in the config which creates new rollouts -> event for each pod
+	{
+		Regexp:    regexp.MustCompile(`ns/openshift-route-controller-manager deployment/route-controller-manager - reason/ScalingReplicaSet \(combined from similar events\): Scaled down replica set route-controller-manager-[a-z0-9-]+ to [0-9]+`),
+		TestSuite: stringPointer("openshift/build"),
+	},
 	{
 		Regexp:    regexp.MustCompile(`ns/openshift-controller-manager daemonset/controller-manager - reason/SuccessfulDelete \(combined from similar events\): Deleted pod: controller-manager-[a-z0-9-]+`),
 		BZ:        "https://bugzilla.redhat.com/show_bug.cgi?id=2034984",
```

Comment:
this was originally bug, but I think we could remove the BZ now

Reply:
given the BZ has been closed(as not a bug), we should remove it.

Reply:
removed